src/main/java/frc/robot/subsystems/flywheel/FlywheelCommands.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.subsystems.flywheel;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.flywheel.commands.RotateFlywheel;
+
+public class FlywheelCommands {
+    private final Flywheel flywheel;
+
+    public FlywheelCommands(Flywheel flywheel) {
+        this.flywheel = flywheel;
+    }
+
+    public Command rotate(DoubleSupplier upperRollerSpeedSupplier, DoubleSupplier lowerRollerSpeedSupplier) {
```

Comment:
rotate sounds weird. I would say something with "controller" is better

Reply:
Yeah naming this was hard, and I don't like this name too, but `controller` seems even weirder to me. Think of when writing commands compositions, verbs that actually says what it does are more readable.